apps/prairielearn/src/lib/server-jobs.sql

```diff
@@ -99,3 +99,11 @@ SET
 WHERE
   js.id = $job_sequence_id
   AND js.status = 'Running'::enum_job_status;
+
+-- BLOCK select_job_output
+SELECT
+  j.*
+FROM
+  jobs AS j
+WHERE
+  j.job_sequence_id = $job_sequence_id;
```

Comment:
You should probably throw an `ORDER BY` clause on this query. Here's what we do when fetching job sequences for display on the job sequence page:

https://github.com/PrairieLearn/PrairieLearn/blob/16723e238fdd28d494b9e8bd214c2a65d9d749c6/apps/prairielearn/src/lib/server-jobs-legacy.sql#L282-L284

Reply:
Done in 1ec8018fe61ca7a40da29bb53f75738cf105a90e. This didn't affect the currently only use of this function, which contains a single job in the job sequence, but I agree that future-proofing it makes sense here.